metadata-ingestion/tests/integration/athena/athena_mce_golden.json

```diff
@@ -273,7 +273,7 @@
                         },
                         "fields": [
                             {
-                                "fieldPath": "[version=2.0].[type=string].employee_id",
+                                "fieldPath": "employee_id",
```

Comment:
IIRC we discussed already about this field path v1 vs v2 breaking change, right? 
how is this is happening here again?